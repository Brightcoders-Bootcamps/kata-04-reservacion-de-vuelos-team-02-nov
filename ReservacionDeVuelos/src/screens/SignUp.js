import React from 'react';
import {View} from 'react-native';
import {styles} from '../styles/SignUpStyles';
import Form from '../components/molecules/SignUpForm';
import SignUpTitle from '../components/atoms/SignUpAtoms/SignUpTitle';
import SignUpButtons from '../components/atoms/SignUpAtoms/SignUpButtons';
const SignUp = () => {
  return (
    <View style={styles.SignUpContainer}>
      <SignUpTitle />
      <Form />
      <SignUpButtons />
    </View>
  );
};

export default SignUp;
