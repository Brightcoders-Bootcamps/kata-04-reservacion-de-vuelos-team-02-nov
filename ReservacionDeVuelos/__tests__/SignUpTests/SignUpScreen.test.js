import React from 'react';
import SignUp from '../../src/screens/SignUp';
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<SignUp />).toJSON();
  expect(tree).toMatchSnapshot();
});
