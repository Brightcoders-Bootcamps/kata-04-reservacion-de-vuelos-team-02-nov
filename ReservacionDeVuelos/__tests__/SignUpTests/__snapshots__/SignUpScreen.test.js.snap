// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly snapshot 1`] = `ShallowWrapper {}`;
